arbitrum-docs/node-running/how-tos/data-availability-committee/configure-the-dac-in-your-chain.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'How to configure a Data Availability Committee: configure the DAC in your chain'
+description: This how-to will help you configure the DAC in your chain.
+author: jose-franco
+sidebar_label: Configure the DAC in your chain
```

Comment:
Modified the sidebar to "How to configure the Data Availability Committee (DAC) in your chain" since this document is specific about one step of the process and having the same title as the section might be confusing for users. I kept the suggested convention.